### PR TITLE
update aspire versions

### PR DESCRIPTION
**Pull request by vhvb1989**
Update .NET Aspire version to 8.2.0 for testing

### CODE REVIEW

**Review by ellismg — APPROVED** [review]

File: cli/azd/test/functional/aspire_test.go [hunk 1]
```diff
@@ -235,9 +235,6 @@ func Test_CLI_Aspire_DetectGen(t *testing.T) {
 
 // Test_CLI_Aspire_Deploy tests the full deployment of an Aspire project.
 func Test_CLI_Aspire_Deploy(t *testing.T) {
-	if cfg.CI && os.Getenv("AZURE_RECORD_MODE") != "live" {
-		t.Skip("skipping live test")
-	}
```
Comment (vhvb1989) [inline cli/azd/test/functional/aspire_test.go:240]: @weikanglim do we not have recordings here anymore?

Comment (weikanglim) [reply cli/azd/test/functional/aspire_test.go:240]: We never had recordings for this, was slightly worried about overall flakiness/usefulness

**Comment by vhvb1989** [comment]
taking Windows and Mac passing to move on and unblock CI

**Pull request merged**